corehq/apps/reports/views.py

```diff
@@ -1137,17 +1151,19 @@ def get_scheduled_report_response(couch_user, domain, scheduled_report_id,
         email,
         attach_excel=attach_excel,
         lang=notification.language,
-        report_id=scheduled_report_id
     )
 
+    return render_full_report_notification(request, content, user_email, notification), excel_files
+
 
 def _render_report_configs(request, configs, domain, owner_id, couch_user, email,
```

Comment:
would be good to add a docstring here to say that this only renders the report body